### ISSUE TITLE
tagbot not working

### ISSUE THREAD

**Issue by johnnychen94** [cjdoris/Bokeh.jl#9]
CRef: https://github.com/JuliaRegistries/TagBot/issues/239

**Comment by cjdoris** [comment]
Thanks for the info, I'll see what happens next time I make a release.

On a related note, CompatHelper stopped working a few days ago and I don't know why. Do you think it's the same issue?

**Comment by johnnychen94** [comment]
Yeah I think that's the same reason -- GitHub now requires us to add explicit permission to CI configuration.
